test/0.1.1-rc.17/util.ts

```diff
@@ -100,6 +100,13 @@ export async function writeProposalIPFS(data: any) {
   return ipfsResponse[0].path;
 }
 
+export async function writeToIPFS(data: string) {
```

Comment:
We already have this though (but with stringify, does it mater?): https://github.com/daostack/subgraph/blob/a3a63320d93e513782700fd62c3d65288f62db31/test/0.1.1-rc.17/util.ts#L96

Reply:
`writeProposalIPFS`  this is doing JSON.stringify ...which is different.

Reply:
ok